html/pfappserver/root/src/views/Configuration/adminRoles/_components/TheSearch.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <b-card no-body>
+    <b-card-header>
+      <h4 class="d-flex align-items-center mb-3">
+        {{ $t('Admin Roles') }}
+        <base-button-help class="text-black-50 ml-1" url="PacketFence_Installation_Guide.html#_admin_access" />
+      </h4>
+      <p class="mb-0" v-t="'Define roles with specific access rights to the Web administration interface. Roles are assigned to users depending on their authentication source.'"></p>
+    </b-card-header>
+    <div class="card-body">
+      <base-search :use-search="useSearch">
+        <b-button variant="outline-primary" @click="goToNew">{{ $t('New Admin Role') }}</b-button>
+      </base-search>
+      <b-table ref="tableRef"
+        :busy="isLoading"
+        :hover="items.length > 0"
+        :items="items"
+        :fields="visibleColumns"
+        :sort-by="sortBy"
+        :sort-desc="sortDesc"
+        @sort-changed="setSort"
+        @row-clicked="goToItem"
+        class="mb-0"
+        show-empty
+        no-local-sorting
+        sort-icon-left
+        fixed
+        striped
+        selectable
+        @row-selected="onRowSelected"
+      >
+        <template v-slot:empty>
+          <slot name="emptySearch" v-bind="{ isLoading }">
+            <base-table-empty :is-loading="isLoading">{{ $t('No results found') }}</base-table-empty>
+          </slot>
+        </template>
+        <template #head(selected)>
+          <span @click.stop.prevent="onAllSelected">
+            <template v-if="selected.length > 0">
+              <icon name="check-square" class="bg-white text-success" scale="1.125" />
+            </template>
+            <template v-else>
+              <icon name="square" class="border border-1 border-gray bg-white text-light" scale="1.125" />
+            </template>
+          </span>
+        </template>
+        <template #cell(selected)="{ index, rowSelected }">
+          <span @click.stop="onItemSelected(index)" style="padding: 12px;">
```

Comment:
Instead of this inline `padding`, apply the `text-center` class to the outer cell.

Reply:
It's not possible to add a class attribute to the `<template>` slot. Furthermore, centered text won't adjust the vertical spacing.

Reply:
Specify the class in the [fields](https://bootstrap-vue.org/docs/components/table#field-definition-reference) variable of the `b-table`.